external_services/lti.py

```diff
@@ -63,7 +63,10 @@ def __init__(self, service, user, instance, request, title, context_id=None, lin
             "launch_presentation_return_url": request.scheme + '://' + request.get_host() + instance.get_absolute_url(),
 
             "tool_consumer_instance_guid": request.get_host() + "/aplus",
-            "tool_consumer_instance_name": "A+ LMS",
+            "tool_consumer_instance_name": settings.BRAND_NAME + " LMS",
+            "tool_consumer_instance_description": settings.WELCOME_TEXT,
```

Comment:
I think we should add a new field to the settings for this. For consideration, we have added following data to HAKA.

```
DisplayName/en: plus.cs.aalto.fi, virtual learning environment
DisplayName/sv: plus.cs.aalto.fi, virtual provmiljö
DisplayName/fi: plus.cs.aalto.fi, virtuaalinen oppimisympäristö
Description/sv: Virtual lärmiljö vid Aalto-universitet institutionen för datateknik
Description/en: Virtual Learning Environment at Aalto University Department of Computer Science
Description/fi: Aalto-yliopiston Tietotekniikan laitoksen virtuaalinen oppimisympäristö
```

I think the later strings would be good here, but we don't have them in the settings.

Reply:
I assume you mean we should use these instead of the WELCOME_TEXT?

Reply:
Yes, except those exact strings are specific to plus.cs.aalto.fi. `settings.py` should include some generic defaults and then the admin would write installation specific strings to local_settings. However, I'm not sure yet what we should do about translations. It's possible that IMS Global has not considered that.

Reply:
Ok, but isn't "A+" specific to plus.cs.aalto.fi too? Should I set it to just something like "Virtual Learning Environment"?

Reply:
Yes, "A+" is specific to plus.cs, but we can use it as a default value. The advantage of using Django settings is that they can be easily overridden on another site/installation.